grid2op/Reward/LinesCapacityReward.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2019-2020, RTE (https://www.rte-france.com)
+# See AUTHORS.txt
+# This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0.
+# If a copy of the Mozilla Public License, version 2.0 was not distributed with this file,
+# you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+# This file is part of Grid2Op, Grid2Op a testbed platform to model sequential decision making in power systems.
+
+import numpy as np
+from grid2op.Reward.BaseReward import BaseReward
+from grid2op.dtypes import dt_float
+
+class LinesCapacityReward(BaseReward):
```

Comment:
what is the difference between that and l2rpn rewards ? Why having 2 rewards that does apparently the same thing ?

Reply:
This is linear and takes into account disconnected lines. I dont mind `rm -rf` the file if thats considered a duplicate